PokemonViewer/PokemonTableViewCell.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  PokemonTableViewCell.swift
+//  PokemonViewer
+//
+//  Created by Claire Chung on 2024/4/19.
+//
+
+import UIKit
+import Combine
+
+class PokemonTableViewCell: UITableViewCell {
+    var url: URL? {
+        didSet {
+            if let url = url {
+                fetchDetailData(from: url)
+            }
+        }
+    }
+    
+    let cellImageView = UIImageView()
+    let nameLabel = UILabel()
+    private var cancellables = Set<AnyCancellable>()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupViews()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupViews() {
+        addSubview(nameLabel)
+        addSubview(cellImageView)
+        
+        nameLabel.translatesAutoresizingMaskIntoConstraints = false
+        cellImageView.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            nameLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 10),
+            nameLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 20),
+            nameLabel.widthAnchor.constraint(equalTo: contentView.widthAnchor, constant: 100),
+            cellImageView.leadingAnchor.constraint(equalTo: nameLabel.trailingAnchor, constant: 10),
+        ])
```

Comment:
```suggestion
        NSLayoutConstraint.activate([
            nameLabel.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 10),
            nameLabel.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 20),
            nameLabel.widthAnchor.constraint(equalTo: contentView.widthAnchor, constant: 100),
            cellImageView.leadingAnchor.constraint(equalTo: nameLabel.trailingAnchor, constant: 10),
            cellImageView.topAnchor.constraint(equalTo: contentView.topAnchor),
            cellImageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)
        ])
```

我直接在 Github 上改的，所以可能有打錯，再自己修正一下